Mage.Client/src/main/java/mage/client/game/GamePanel.java

```diff
@@ -1610,14 +1617,27 @@ private void initComponents() {
         txtHoldPriority.setToolTipText("Holding priority after the next spell cast or ability activation");
         txtHoldPriority.setVisible(false);
 
-        btnToggleMacro = new KeyboundButton(KEY_CONTROL_TOGGLE_MACRO);
-        btnCancelSkip = new KeyboundButton(KEY_CONTROL_CANCEL_SKIP); // F3
-        btnSkipToNextTurn = new KeyboundButton(KEY_CONTROL_NEXT_TURN); // F4
-        btnSkipToEndTurn = new KeyboundButton(KEY_CONTROL_END_STEP); // F5
-        btnSkipToNextMain = new KeyboundButton(KEY_CONTROL_MAIN_STEP); // F7
-        btnSkipStack = new KeyboundButton(KEY_CONTROL_SKIP_STACK); // F10
-        btnSkipToYourTurn = new KeyboundButton(KEY_CONTROL_YOUR_TURN); // F9
-        btnSkipToEndStepBeforeYourTurn = new KeyboundButton(KEY_CONTROL_PRIOR_END); // F11
+        // Some themes don't have button text
+        if (!this.currentTheme.isSkipShortcutsVisible()) {
+            btnToggleMacro = new JButton();
+            btnCancelSkip = new JButton();
+            btnSkipToNextTurn = new JButton();
+            btnSkipToEndTurn = new JButton();
+            btnSkipToNextMain = new JButton();
+            btnSkipStack = new JButton();
+            btnSkipToYourTurn = new JButton();
+            btnSkipToEndStepBeforeYourTurn = new JButton();
```

Comment:
Looks fine for me:

![Capture](https://user-images.githubusercontent.com/13023067/86253838-55bac680-bbb5-11ea-8a19-46bb98c95fe1.PNG)

Perhaps this yellow border is also drawn somewhere else, because it indeed shouldn't work. ;)

Reply:
yup, check for example btnToggleMacroActionPerformed method. it sets border for clicked button. Should we stay with border solution or tint in KeyboundButton?

Reply:
I found what's tint for -- it's toggle button gray on mouse press, not border (e.g. indicate that user really clicks on buttons). F7 example:

![shot_200701_184127](https://user-images.githubusercontent.com/8344157/86257397-86592b00-bbca-11ea-9e4b-bbf558f85f52.png)

So it's broken with current theme code.